### DISCUSSION TITLE
Wiki js - storage options

### DISCUSSION THREAD

**Discussion by abockova-accenture** [requarks/wiki#6432] [Help / Questions] [answered]
Hello,

currently we have PostgreSQL database which collects all data. Is it sustainable to have the assets in the database? Images can be several Mo each. Or would it be better, if possible, to store the assets in a storage account?
Thank you for your answer.

**Comment by NGPixel** [comment, marked as answer]
Images shouldn't be a problem. But larger files should be stored in a proper blob storage (AWS S3, Azure Blob Storage, DO Spaces, etc.)
